appExposeOverview.js

```diff
@@ -58,9 +58,8 @@ var AppExposeOverview = new Lang.Class({
 
 		// Only show application windows in thumbnails
 		const originalThumbnailFunction = this.originalOverviewFunction;
-		WorkspaceThumbnail.WorkspaceThumbnail.prototype._isOverviewWindow = function(win) {
-			const originalResult = originalThumbnailFunction(win);
-			const metaWindow = win.get_meta_window();
+		WorkspaceThumbnail.WorkspaceThumbnail.prototype._isOverviewWindow = function(metaWindow) {
+			const originalResult = originalThumbnailFunction(metaWindow);
```

Comment:
This was actuallly right before and this breaks the thumbnails filter (not thumbnail shown), so it should be reverted.

Reply:
actually, I have no idea how `metaWindow` is different to `win` - I merely made some changes to the original code by @ggcode as far as I understood it.
Trying to learn about the GnomeShell API, hwoever, only [led me here](https://www.reddit.com/r/gnome/comments/f2qhrx/gnome_extensions_api/).

I just want to note, that you can directly push changes to my fork as you think fit. Also, maybe ggcode can help out.